apps/backend/app.py

```diff
@@ -156,21 +190,13 @@ def frange(start, stop, step=None):
         count += 1
 
 def gen_list(otherVar, paramspos):
-    try:
-        step = otherVar['step']
-        if step == "":
-            if(otherVar['type'] == 'integer'):
-                step = 1
-            elif(otherVar['type'] == 'float'):
-                step = 0.1
-    except KeyError:
-        print('OtherVar did not have type attribute')
-
     if otherVar['type'] == 'integer':
+        step = 1 if otherVar['step'] == '' else otherVar['step']
```

Comment:
python ternary statement? 😳

Should probably extract the default steps to be named constants

Reply:
Python Ternary Statement 😎, also will add named constants